cmd/qx/qx.go

```diff
@@ -167,15 +170,19 @@ func main() {
 	diffToHashrateCmd := flag.NewFlagSet("diff-to-gps", flag.ExitOnError)
 	diffToHashrateCmd.IntVar(&edgeBits, "e", 24, "edgebits")
 	diffToHashrateCmd.IntVar(&blocktime, "t", 15, "blocktime")
-	diffToHashrateCmd.IntVar(&blocktime, "m", 1, "mheight")
+	diffToHashrateCmd.IntVar(&mheight, "m", 1, "mheight")
+	diffToHashrateCmd.StringVar(&network, "n", "testnet", "decode rawtx for the target network. (mainnet, testnet, privnet,mixnet)")
+	diffToHashrateCmd.StringVar(&powType, "p", "cuckaroo", "the target cuckoo pow. ( cuckaroo, cuckatoo)")
 	diffToHashrateCmd.Usage = func() {
 		cmdUsage(diffToHashrateCmd, "Usage: qx diff-to-gps -e 24 -t 15 [difficulty uint64]\n")
 	}
 
 	gpsToDiffCmd := flag.NewFlagSet("gps-to-diff", flag.ExitOnError)
 	gpsToDiffCmd.IntVar(&edgeBits, "e", 24, "edgebits")
 	gpsToDiffCmd.IntVar(&blocktime, "t", 15, "blocktime")
-	gpsToDiffCmd.IntVar(&blocktime, "m", 1, "mheight")
+	gpsToDiffCmd.IntVar(&mheight, "m", 1, "mheight")
+	gpsToDiffCmd.StringVar(&network, "n", "testnet", "decode rawtx for the target network. (mainnet, testnet, privnet,mixnet)")
```

Comment:
`decode rawtx for the target network` ？please be very careful about any modification you added included msg display or comments.  be careful about the copied&pasted code